src/electron-main.ts

```diff
@@ -119,8 +119,9 @@ async function tryPaths(name: string, root: string, rawPaths: string[]): Promise
 
 const homeserverProps = ["default_is_url", "default_hs_url", "default_server_name", "default_server_config"] as const;
 
-// Find the webapp resources and set up things that require them
-async function setupGlobals(): Promise<void> {
+async function loadConfig(): Promise<void> {
+    if (global.vectorConfig) return;
+
     // find the webapp asar.
     asarPath = await tryPaths("webapp", __dirname, [
```

Comment:
the comment at line 48 says this is set in setupGlobals, which is maybe a bit misleading.

(maybe this logic should move into a separate `getAsarPath()` function which initialises the `asarPath` global on its first call?)